resource-loader.ts

```diff
@@ -0,0 +1,30 @@
+///<reference path=".d.ts"/>
+"use strict";
+
+import path = require("path");
+
+export class ResourceLoader implements IResourceLoader {
+	constructor(private $fs: IFileSystem,
+		private $staticConfig: Config.IStaticConfig) { }
+
+	resolvePath(resourcePath: string): string {
+		return path.join(__dirname, "../../resources", resourcePath);
+	}
+
+	openFile(resourcePath: string): any {
```

Comment:
This function should return a ReadStream. You may also fix the code in file-system.ts